processor/src/operations/crypto_ops.rs

```diff
@@ -143,25 +137,25 @@ where
         let new_node =
             [self.stack.get(13), self.stack.get(12), self.stack.get(11), self.stack.get(10)];
 
-        // update the leaf at the specified index in the merkle set specified by the old root, and
-        // get a Merkle path to the specified leaf. the length of the returned path is expected to
-        // match the specified depth.
-        // TODO: in the future, we should be able to replace sub-trees and not just the leaves,
+        // update the node at the specified index in the Merkle tree specified by the old root, and
+        // get a Merkle path to it. the length of the returned path is expected to match the
+        // specified depth.
+        // TODO: in the future, we should be able to replace sub-trees and not just the nodes,
```

Comment:
We should remove the `TODO` here as this functionality is now enabled.

Reply:
Fixed and added a comment explaining the incremented functionality